keyboards/kopibeng/mnk65_stm32/mnk65.h

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2021 Samuel Lu
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+/*
+ * 
+ *             ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐  ┌───────┐
+ *             │00 │01 │02 │03 │04 │05 │06 │07 │08 │09 │0A │0B │0C │1D │0D │0E │  │0D     │ Full Backspace
+ *             ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┼───┤  └─┬─────┤
+ *             │10   │11 │12 │13 │14 │15 │16 │17 │18 │19 │1A │1B │2C │1C   │1E │    │2D   │
+ *  2.25u      ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤  ┌─┴─┐   │ ISO
+ *  LShift     │20    │21 │22 │23 │24 │25 │26 │27 │28 │29 │2A │2B │2D      │2E │  │1C │   │
+ * ┌────────┐  ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤  └───┴───┘
+ * │30      │  │30  │31 │32 │33 │34 │35 │36 │37 │38 │39 │3A │3B │3C    │3D │3E │
+ * └────────┘  ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+ *             │40  │41  │42  │46                      │4A  │4B  │ │4C │4D │4E │
+ *             └────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
+ *             ┌────┬────┬────┬────────┬────┬──────────┬────┬────┐
+ *             │40  │41  │42  │44      │46  │48        │4A  │4B  │ 2.25u-1.25u-2.75u Split
+ *             └────┴────┴────┴────────┴────┴──────────┴────┴────┘
+ *             ┌────┬────┬────┬──────────┬────┬────────┬────┬────┐
+ *             │40  │41  │42  │44        │46  │48      │4A  │4B  │ 2.75u-1.25u-2.25u Split
+ *             └────┴────┴────┴──────────┴────┴────────┴────┴────┘
+ *             ┌─────┬───┬─────┬───────────────────────────┬─────┐
+ *             │40   │41 │42   │46                         │4B   │ 7u Tsangan
+ *             └─────┴───┴─────┴───────────────────────────┴─────┘
+ *
+ */
+
+// LAYOUT_all:
+#define LAYOUT_all( \
```

Comment:
If this is the only layout included here, then it should be just `LAYOUT`.
```suggestion
#define LAYOUT( \
```

However, you should probably break out the different layouts.